cibuildwheel/options.py

```diff
@@ -139,7 +140,7 @@ def _dig_first(*pairs: Tuple[Mapping[str, Setting], str], ignore_empty: bool = F
 
             return value
 
-    raise KeyError(key)
+    raise KeyError(key)  # pylint: disable=undefined-loop-variable
```

Comment:
This line did trip me up on first reading it too, maybe this would avoid the comment?

```suggestion
    last_key = pairs[-1][1]
    raise KeyError(last_key)
```

Reply:
Oh, nice. :)